src/store/plugins/observables.js

```diff
@@ -25,7 +25,7 @@ export default store => {
     entityType,
     ...other,
     sender,
-    chainName: store.state.chainNames[sender],
+    chainName: store.state.chainNames && store.state.chainNames[sender],
```

Comment:
```suggestion
    chainName: store.state.chainNames?.[sender],
```
this is not the same as https://github.com/aeternity/superhero-wallet/pull/562#discussion_r526400981